module-system/module-implementations/sov-nft-module/src/offchain.rs

```diff
@@ -0,0 +1,168 @@
+#[cfg(feature = "offchain")]
+use postgres::NoTls;
+use sov_modules_macros::offchain;
+
+#[cfg(feature = "offchain")]
+use crate::utils::get_collection_address;
+use crate::{Collection, CollectionAddress, Nft};
+
+// CREATE TABLE collection (
+//     collection_address TEXT PRIMARY KEY,
+//     collection_name TEXT NOT NULL,
+//     creator_address TEXT NOT NULL,
+//     frozen BOOLEAN NOT NULL,
+//     metadata_url TEXT,
+//     supply BIGINT NOT NULL
+// );
```

Comment:
nit: Let's remove commented code or add some explanation of what it's doing here.